src/app/home/home.component.html

```diff
@@ -1,38 +1,25 @@
 <main class="main-div">
-    <section class="main-section-1">
-        <div class="card">
-            <div class="card-header">
-                Fazer publicação
-            </div>
-            <div class="card-body">
-                <div class="mb-3">
-                    <input class="form-control" type="text" placeholder="O que está cozinhando hoje?"
-                        aria-label="default input example">
-                    <textarea class="form-control" id="exampleFormControlTextarea1" rows="3"
-                        placeholder="Deixe um comentário sobre a receita."></textarea>
-                    <input class="form-control" type="file" id="formFile">
-                </div>
-                <a href="#" class="btn btn-primary">Publicar</a>
+
+    <div class="title-block">
+        <h1 id="title-page">Find meals by your ingredient</h1>
+        <p id="frase">A recipe has no soul. You, as the cook, must bring soul to the recipe.<br>-Thomas Keller</p>
+    </div>
+
+
+    <section class="search-section">
+            <div class="search-block">
+                    <input id="search-input" type="text" [formControl]="search" class="form-control" placeholder="Ex: chicken, banana, meat...">
+                    <button (click)="sendMessage()" class="btn btn-primary">Search</button>
             </div>
-        </div>
     </section>
-    <section class="main-section-2">
-        <div *ngIf="receita">
-            <div class="card" *ngFor="let i of receita.meals">
-                <div class="card-header title-card">{{ i.strMeal }}</div>
-                <img [attr.src]="i.strMealThumb" class="card-img-top" alt="...">
+    <section class="recipe-section" *ngIf="recipes">
+            <div class="card card-style" *ngFor="let i of recipes">
```

Comment:
Aqui eu alteraria o nome da variavel, para fazer mais sentido:
```suggestion
            <div class="card card-style" *ngFor="let recipe of recipes">
```

Mas de novo, não precisa alterar nesse PR, pode deixar para o futuro.

Reply:
Beleza, altero sim. Eu tinha colocado "i" para não ficar tão grande o tamanho da variavel que fosse chamar através do "i".

Reply:
Sim sim, entendo, mas é uma boa partica ter sempre nomes descritivos mesmo que o nome da variavel seja longo.

Por exemplo:
```js
const a = 4;
const numberOfCardsToDisplay = 4;
```
Eu não faço ideia do que significa `a` no código, eu teria que procurar e demoraria para talvez entender, mas a segunda variável já diz o que significa.